client/blocks/jitm/index.jsx

```diff
@@ -191,8 +199,4 @@ const mapStateToProps = ( state, { messagePath } ) => {
 	};
 };
 
-const mapDispatchToProps = {
-	recordTracksEvent,
-};
```

Comment:
Removing this `mapDispatchToProps` and replacing it with a `dispatch( ... )` call. The surrounding code is already using `dispatch` from `useDispatch`.